### PR TITLE
ToCode Solution: תרגול: מבוא לשורת הפקודה

### PR DESCRIPTION
**Pull request by Yulia88**

### CODE REVIEW

File: 07_cmd_lab/cmdlab.txt [hunk 1]
```diff
@@ -1,15 +1,43 @@
...
+bash
 
-2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
+2. View the file /etc/passwd on your machine with the command "cat /etc/passwd".
+ Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
 
-3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+it's a password file. holds per each user one line and 7 fields :
+login name 
+optional encrypted password
```
Comment (ynonp) [inline 07_cmd_lab/cmdlab.txt:28]: Optional only in theory :) In practice you won't find anyone's encrypted password there for the past 30 years or so. Modern practice keeps all users encrypted passwords in the file `/etc/shadow`

File: 07_cmd_lab/cmdlab.txt [hunk 2]
```diff
@@ -1,15 +1,43 @@
...
+user name or comment field
+user home directory
+optional user interpreter 
+
+
+3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+
+
+ls -l --block-size /etc/
+
```
Comment (ynonp) [inline 07_cmd_lab/cmdlab.txt:39]: Did it work? I tried and got the error:

```
 invalid --block-size argument '/etc'
```

block size controls how ls prints the size column, for example it lets you print the size in bytes, kilo-bytes, MB etc. 

your task is to find another option that sorts according to the file size, but also as a bonus you can try to get block-size to work

File: 07_cmd_lab/cmdlab.txt [hunk 3]
```diff
@@ -1,15 +1,43 @@
...
 
 4. Print current date/time formatted as: "Sun 21/03/2010, 14:30". Hint: read "man date" and search for the correct format string.
+
+date +'%a %m/%d/%Y, %R'
```
Comment (ynonp) [inline 07_cmd_lab/cmdlab.txt:43]: 👍

**Review by ynonp — COMMENTED** [review]
Hi Yulia,
Great work! Now you only need to find how to get `ls` to sort files by size and move on to the next exercise

File: 07_cmd_lab/cmdlab.txt (hunk 2 above)
Comment (Yulia88) [reply 07_cmd_lab/cmdlab.txt:39]: I mistakenly wrote the wrong command
Block size: 
ls -l --block-size=M /etc/

The command that was in the question is:
ls -l -S /etc/

**Comment by ynonp** [comment]
Fantastic. Closing.

**Pull request closed without merging**